src/wazuh_db/wdb_global.c

```diff
@@ -1321,9 +1326,34 @@ int wdb_global_groups_number_get(wdb_t *wdb, int agent_id) {
     return groups_number;
 }
 
-wdbc_result wdb_global_validate_groups(wdb_t *wdb, cJSON *j_groups, int agent_id) {
+w_err_t wdb_global_validate_group_name(const char *group_name) {
+    const char *current_directory = ".";
+    const char *parent_directory = "..";
+    w_err_t result = OS_SUCCESS;
```

Comment:
This variable is only used to return the value at the end and is never assigned. Just return `OS_SUCCESS` at the end and remove this declaration.

Reply:
You're right, didn't see that